student_class.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
```

Comment:
- [ ] To avoid errors and for proper assignment of arguments to the parent class, you still need to re-arrange your arguments here, to match what you defined in the Person class. `age` should come before `name`.

  ```
  super(age, name, parent_permission: parent_permission)
  ```